runtime/doc/api.txt

```diff
@@ -9,8 +9,7 @@ Nvim API							   *API* *api*
 Nvim exposes a powerful API that can be used by plugins and external processes
 via |msgpack-rpc|, Lua and VimL (|eval-api|).
 
-Nvim can also be embedded in C applications as libnvim, so the application
-can control the embedded instance by calling the C API directly.
+Applications can also embed libnvim to control via the C ABI.
```

Comment:
There's an object missing in `... to control via the C ABI.`.  Also, API was changed to ABI, which doesn't seem correct.